src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -258,7 +257,7 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
         from: maybeFromUTXOAccountAddress,
         to: quote.inbound_address,
         memo: supportedEvmChainIds.includes(chainId as KnownChainIds)
-          ? utils.hexlify(utils.toUtf8Bytes(memoUtf8))
+          ? toHex(stringToBytes(memoUtf8))
```

Comment:
should be able to handle utf8 directly
https://viem.sh/docs/utilities/toHex#usage

```suggestion
          ? toHex(memoUtf8)
```